ledger/src/blocktree.rs

```diff
@@ -1478,6 +1502,10 @@ fn get_slot_meta_entry<'a>(
     })
 }
 
+fn get_last_hash<'a>(iterator: impl Iterator<Item = &'a Entry> + 'a) -> Option<Hash> {
```

Comment:
Just call this `get_blockhash()`?  That's essentially what the last hash is

Reply:
Haha, I had that, until I realized nothing about my iterator of Entries param guarantees that it is a block.

Reply:
hmm, wait you're right.  A slot can have multiple blockhashes so the current name is better